x-pack/plugins/ml/public/file_datavisualizer/components/import_summary/_import_sumary.scss

```diff
@@ -0,0 +1,19 @@
+.import-summary-list {
```

Comment:
Seems to be a duplicate of `.analysis-summary-list` in `_analysis_summary.scss`, maybe this can be consolidated.

Reply:
i'd quite like to keep these styles separate as it keeps the components independent.
Also, I can see in the future the title width in this component needing to be increased once we get some real user indices being created with long names.